questions.json

```
@@ -41,39 +41,39 @@
   },
   {
     "question": "OO란 CPU가 한 번에 처리 가능한 데이터의 크기를 말합니다. 예를 들어 32 bit 컴퓨터(x86)는 OO의 크기가 32bit, 64 bit 컴퓨터(x64)는 OO의 크기가 64bit입니다.",
-    "choices": [],
+    "choices": ["워드(Word)", "블록(Block)", "세그먼트(Segment)", "페이지(Page)"],
     "correct": "워드(Word)",
     "explanation": "",
     "sources": [],
     "tags": ["CS"]
   },
   {
     "question": "hash collosion을 다루는 방법의 대표적인 예시로는 Open Addressing 방식과 Separate Chaining 방식이 있습니다. Java의 HashMap은 OOOO 방식을 채택하고 있고, 이 방법은 hash collision 발생시 충돌된 버켓은 linked list로 값을 채워 넣습니다\n각 빈칸에 알맞은 키워드를 넣으세요",
-    "choices": [],
+    "choices": ["Separate Chaining", "Open Addressing"],
     "correct": "Separate Chaining",
     "explanation": "",
     "sources": [],
     "tags": ["DS", "Java"]
   },
   {
-    "question": "CPU 내부 구성요소로는 레지스터, ALU, 제어장치가 존재합니다. OOOO는 연산할 수 있는 값들을 임시 저장하는 역할, OOOO는 연산하는 역할, OOOO는 명령어를 해독하고 제어 신호를 보내는 역할을 합니다.",
+    "question": "CPU 내부 구성요소로는 레지스터, ALU, 제어장치가 존재합니다. 그중 OOOO는 연산할 수 있는 값들을 임시 저장하는 역할을 합니다.",
     "choices": ["레지스터", "ALU", "제어장치"],
-    "correct": "레지스터,ALU,제어장치",
+    "correct": "레지스터",
     "explanation": "",
     "sources": [],
     "tags": ["CS"]
   },
   {
     "question": "Java 21에 등장한 OOOO는 경량 스레드라고도 불리우며, 작업하는 스레드에서 Blocking이 발생하면 내부적으로 다른 OOOO를 호출해 작업하도록 만들어 리소스 낭비를 줄이도록 합니다.",
-    "choices": [],
+    "choices": ["가상스레드(Virtual thread)", "가상 메모리(Virtual Memory)", "코루틴(Coroutine)"],
```

```suggestion
    "choices": ["가상스레드(Virtual Thread)", "가상 메모리(Virtual Memory)", "코루틴(Coroutine)"],
```

위에서 언급한 스타일 가이드에 해당하는 부분입니다.